.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -17,27 +17,30 @@
        - [ ] arm64 - `Build Tarball Debian9-Offline_Artifacts/dotnet-smoke-test-prereqs.6.0.xxx.tar.gz`
-     - [ ] Retrieve `microsoft.net.runtime.monoaotcompiler.task` package from internal MSFT feed. [Automation tracking issue](https://github.com/dotnet/source-build/issues/2774) for this step.
-     - [ ] Create new tarball of unique packages
-       - [ ] Extract x64 tarball
-       - [ ] Add `microsoft.net.runtime.monoaotcompiler.task` package to x64 packages
-       - [ ] Extact arm64 tarball
-       - [ ] Copy four `*linux-arm64*` packages to x64 packages
-       - [ ] Create new `dotnet-smoke-test-prereqs.6.0.xxx.tar.gz` tarball
-1. - [ ] [Internal] Upload smoke-test-prereqs tarball to dotnetclimsrc storage account.
-1. - [ ] Notify partners of release.  Include info about how certain we are that this will be the final Microsoft build.
-     - [Internal] Send the dotnetclimsrc tarball links to partners.
+     - [ ] Retrieve additional packages from internal MSFT feed using [this project](https://gist.github.com/lbussell/47a3953686c218ede865e305478df74a).
+     - [ ] Create a new tarball of unique packages using [this script](https://gist.github.com/lbussell/5789974491e3d3ed737aac0e8b97b594).
+     - [ ] Upload smoke-test-prereqs tarball to dotnetclimsrc storage account, following the pattern of previous releases for directory and filename.
          - Never overwrite a tarball. At least change the blob storage virtual dir to represent a new build. This can help avoid timing issues and make it more obvious if stale links were accidentally re-sent rather than new ones.
-     - [Non-Internal] Send the dotnet/installer commit sha along w/link to publicly built source tarball.  Link to the public instructions for building source-build.
+1. - [ ] Update dotnet-security-partners
+     - [ ] Submit a PR to the appropriate `release/*` branch on [`dnceng/security-partners-dotnet`](https://dev.azure.com/dnceng/internal/_git/security-partners-dotnet). A branch was automatically created with the source-build-pre-release pipeline, you just have to submit the PR. ([automation tracking issue for this step](https://github.com/dotnet/source-build/issues/3069))
+     - [ ] squash merge the PR once CI finishes successfully.
+1. - [ ] Run the [source-build-release-mirror](https://dev.azure.com/dnceng/internal/_build?definitionId=1221&_a=summary) pipeline
+     - **Important**: this pipeline should be ran only once per hand-off. It mirrors and tags all of the branches listed in the parameters of `source-build-release-mirror.yml`. If you need to mirror and tag only some branches/repos, you can edit the yaml object directly at pipeline queue time.
+     - **Important**: replace the `sdkVersion` with the new SDK version (e.g. 6.0.112) in the "mirrors" parameter for each branch.
+1. - [ ] Notify partners of release. Send one email for all releases. Include the following in your email:
```

Comment:
Do we and if not should we, have a tracking issue for automation to create the email.  At least a tool that will generated the body of the email.

Reply:
I filed the issue and added the note to the readme.